components/pdf_preview.jsx

```diff
@@ -40,14 +38,19 @@ export default class PDFPreview extends React.PureComponent {
             loading: true,
             success: false,
         };
-
-        for (let i = 0; i < MAX_PDF_PAGES; i++) {
-            this[`pdfCanvasRef-${i}`] = React.createRef();
-        }
     }
 
     componentDidMount() {
         this.getPdfDocument();
+        if (this.container.current) {
+            this.parentNode = this.container.current.parentElement.parentElement;
```

Comment:
I think you accidentally removed the code to add the scroll handler to the parent